lib/MultiStats.js

```diff
@@ -70,8 +72,13 @@ class MultiStats {
 		};
 	}
 
+	/**
+	 * @param {*} options stats options
```

Comment:
Bad typing

Reply:
Updated, I put it as `any` now. I don't actually know how the options are determined. Hopefully with more time (or someone with more experience) can properly type this.